.github/workflows/changelog-preview.yml

```
@@ -0,0 +1,57 @@
+name: Changelog Preview
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  previewChangelog:
+    name: Preview Changelog
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    permissions:
      pull-requests: write
```
Otherwise Dependabot PRs will fail (see in LAPIS currently)